db/migrate/20230829001347_add_approved_to_stories.rb

```diff
@@ -0,0 +1,5 @@
+class AddApprovedToStories < ActiveRecord::Migration[7.0]
+  def change
+    add_column :stories, :approved, :boolean, default: nil
```

Comment:
I feel it could be more maintainable and semantic to have a `status` or `state` integer column and then use Rails Enum to handle pending/approved/rejected states.

That would also add some methods to the model like `story.approved?` or`story.pending?` or `story.approve!` to be more declarative and less imperative

Reply:
Agreed